vizro-ai/docs/pages/tutorials/explore-vizro-ai.md

```diff
@@ -42,6 +36,7 @@ Vizro-AI is versatile, supporting prompts and chart visualizations in multiple l
 Subsequently, we'll switch to German and prompt the visualization of life expectancy in the United States over time, comparing it to the global life expectancy trend. For this example, we'll include `explain=True` to obtain comprehensive insights into both the data and the generated code.
```

Comment:
Where is the Chinese example coming from? 😄 Have we double-checked with someone that actually speaks it? haha

Reply:
Lingyi wrote it!